packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -579,4 +593,44 @@ describe('TokenDetectionController', () => {
 
     expect(getBalancesInSingleCallMock.called).toBe(true);
   });
+
+  describe('detectTokens', () => {
+    it('should detect tokens with networkClientId and address params', async () => {
```

Comment:
This test seems to involve more than just `detectTokens` (in fact this method is stubbed out). Does it makes sense to move this to a different describe, perhaps `startPollingByNetworkClientId`?